static-assets/yui/container/container.js

```diff
@@ -5743,10 +5743,10 @@ version: 2.6.0
          * @param {Event} e The DOM event object
          */
         _onElementFocus : function(e){
-
+            var authDialog = document.getElementById('auth-monitor-dialog');
             var target = Event.getTarget(e);
 
-            if (target !== this.element && !Dom.isAncestor(this.element, target) && _currentModal == this) {
+            if (target !== this.element && (!authDialog && !Dom.isAncestor(authDialog, target)) && !Dom.isAncestor(this.element, target) && _currentModal == this) {
```

Comment:
Let's double check this condition...

This doesn't seem to cover when **there is an** auth dialog.

The original condition reads:
- Focused element is not the modal's element AND
- Focused element is not a child of the modal's element AND
- The modal is the topmost modal (current modal) THEN
- Set the focus to an element in this modal

We need to add:
```
AND... (
  There is no auth dialog OR (
    the focused element IS the auth dialog OR 
    the focused element IS a child of the auth dialog
  )
)
```

Reply:
Actually, on second thought, the simple existence of an auth dialog should suffice to skip the focus trap.